pyttb/sptensor.py

```diff
@@ -828,16 +1168,48 @@ def logical_xor(
 
     def mask(self, W: sptensor) -> np.ndarray:
         """
-        Extract values as specified by a mask tensor
+        Extract :class:`pyttb.sptensor` values as specified by `W`, a
+        :class:`pyttb.sptensor` containing only non-zero values that are
+        ones (1). The values in the :class:`pyttb.sptensor` corresponding
+        to the indices of the ones (1) in `W` will be returned as a column
```

Comment:
This might be a little clearer?
```suggestion
        Extract :class:`pyttb.sptensor` values as specified by `W`.
        The values in the :class:`pyttb.sptensor` corresponding
        to entries containing ones (1) in `W` will be returned as a column
```